### DISCUSSION TITLE
Git stuffs cannot open by GitHub Mobile

### DISCUSSION THREAD

**Discussion by Neodevils** [community/community#34109] [Mobile] [answered]
Hello 👋🏻

GitHub Mobile has problem while showing something about related to Git websites. For example, if I try to open Gist via logging to my account it justs suggests me to log in with "Chrome or Mi Browser(phone's default browser)?" And when I just want to check running apps, I see GitHub Mobile is already running but it can not redirect me. And when I check the app; it just shows the home tab. This issue has to be fixed to be honest.

**Comment by Iron-Ham** [comment, marked as answer]
Hi @Neodevils! We don't currently support rendering Gists at this time, but I'll open up an internal issue to track it.

**Reply by Neodevils** [reply]
> Hi @Neodevils! We don't currently support rendering Gists at this time, but I'll open up an internal issue to track it. 

Thanks @Iron-Ham.
